src/App.tsx

```diff
@@ -1,4 +1,4 @@
-import { Routes, Route, Navigate } from 'react-router-dom';
+import { Routes, Route, Navigate, useSearchParams } from 'react-router-dom';
```

Comment:
todo: GA tracking

Reply:
created ticket on this topic: https://www.notion.so/voltz/GA-analytics-for-Referrer-requirments-ca87cc78fabe443e977f2572fe2c74ab @CyclopsRex I think once we have it clear what we are after we can do the impl. right now different people have different ideas imho